data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1845 @@
+# Copyright (c) 2022 by Arachi-Lover
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Heliarch Investigation 1"
+	name "Transport Heliarch Agents"
+	description "Bring <bunks> Heliarch agents to <destination>."
+	minor
+	passengers 3
+	to offer
+		random < 65
+		"coalition jobs" >= 70
+		has "Coalition: Contributor: done"
+		not "Resistance: Questions: done"
+	source
+		near "Quaru" 4 10
+	destination "Ring of Friendship"
+	on offer
+		conversation
+			`A group of Heliarch agents rapidly approach you as you step onto the spaceport, looking rather hasty as they stand in front of you.`
+			`	"Looking for work, the human visitor is?" The Saryd asks.`
```

Comment:
This is perhaps a stylistic choice, but I think that when these guys are talking with their reversed syntax it doesn't need a comma. I think that if an English-speaker said it this way a comma would be used because we are essentially going back and filling in a blank that was missed, but for these guys this is just the way they speak.

"Looking for work the human visitor is?"

I have only mentioned it once because I know that it would be pretty annoying to have a million comments all basically saying the same thing, especially if that one thing has a good chance of being over-ruled. :-)

Reply:
I might have used a lot more commas than usual for the Intros as a good portion of them was written years ago, but Coalition speech in general has a lot of commas used, so won't be dropping most of them.